### DISCUSSION TITLE
Vulkan 1.3 support

### DISCUSSION THREAD

**Discussion by DragonSWDev** [KhronosGroup/MoltenVK#1852] [General]
Hello. Is there any TODO list what is missing for exposing Vulkan 1.3 support?

**Comment by billhollings** [comment]
Not yet. 

For Vulkan 1.2, we created such a list as a [MoltenVK issue](https://github.com/KhronosGroup/MoltenVK/issues/1567).

We haven't done so for Vulkan 1.3 yet, but only because it hasn't been a priority yet.

I'll get to it at some point, but in the meantime, happy to have anyone create an issue if they're up for it.

**Reply by DragonSWDev** [reply]
Thank you for answer. I hope you will be able to get to it soon. Thank you for your work of making open standard work in Apple ecosystem!
